tests/mocks.py

```diff
@@ -0,0 +1,31 @@
+class Dynamic(object):
+    def __init__(self, label, **kwargs):
+        self.label = label
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    def __repr__(self):
+        return str(self.label)
+
+
+def mock_target(id, backup=True, restore=True, **kwargs):
+    return Dynamic(id, id=id, mount_id="mount-{}".format(id),
+                   name="name-{}".format(id),
+                   options=Dynamic("opt", backup=backup, restore=restore),
+                   **kwargs)
+
+
+mock_starport_settings = {
```

Comment:
should these not be starwars themed?  We're mixing canons here :joy: 
![superman-star-wars-1038127](https://user-images.githubusercontent.com/1558093/39353082-8d75a728-49fe-11e8-874b-dd95a13c9064.jpg)